### PR TITLE
Convert ButtonToolBar to RTL

### PR DESCRIPTION
**Pull request by Proful**
Converted ButtonToolBar test cases to RTL

### CODE REVIEW

**Review by JacobMGEvans — APPROVED** [review]
LGTM, as long as checks pass I'd say merge it 😄

File: src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx [hunk 1]
```diff
@@ -17,17 +17,17 @@ describe('Button Tool Bar', () => {
...
   })
 
   it('should return null when there is no button in the provider', () => {
     jest.spyOn(ButtonBarProvider, 'useButtons').mockReturnValue([])
 
-    const wrapper = mount(<ButtonToolBar />)
-
-    expect(wrapper.html()).toBeNull()
+    render(<ButtonToolBar />)
+    expect(screen.queryAllByRole('button')).toHaveLength(0)
```
Comment (JacobMGEvans) [inline src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx]:
```suggestion
    expect(screen.queryAllByRole('button')).not.toBeInTheDocument()
```

Comment (JacobMGEvans) [reply src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx]: This checks for the same thing essentially, however, is more semantically explicit

**Review by JacobMGEvans — CHANGES_REQUESTED** [review]

File: src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx (hunk 1 above)
Comment (Proful) [reply src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx]: Fixed.

**Pull request merged**